packages/SwingSet/src/kernel/loadVat.js

```diff
@@ -187,7 +187,7 @@ export function makeVatLoader(stuff) {
         enableSetup,
         enablePipelining,
         enableInternalMetering: !isDynamic,
-        notifyTermination: metered ? notifyTermination : undefined,
+        notifyTermination,
```

Comment:
Didn't we decide this could be removed? Double check, but I think vatManagers no longer call `notifyTermination` themselves so we can stop giving it to them (and remove it from the "known option names" check in .. maybe `vatManager/factory.js`)

Reply:
We did decide that, but it turns out we were wrong, as I discovered in testing after I took it out.  It's used in the kernel's `addVatManager` function which gets it via the `managerOptions` parameter.  It's possible things could be restructured to avoid this pathway, but it's not a no-brainer.